### DISCUSSION TITLE
How to write a release-only portfile?

### DISCUSSION THREAD

**Discussion by kevin--** [microsoft/vcpkg#32739] [Q&A] [answered]
This issue was filed and closed without any useful information, but I am experiecing the same behavior: https://github.com/microsoft/vcpkg/issues/10410

I am writing a port file for a release-only library similar to the OP. Setting the `VCPKG_BUILD_TYPE` in the port file does not affect anything.

I get the following warning when running vcpkg:

```
Mismatching number of debug and release binaries. Found 0 for debug but 2 for release.
Debug binaries


Release binaries

    /Users/xxx/projects/vcpkg/packages/mylib_arm64-osx/lib/libone.a
    /Users/xxx/projects/vcpkg/packages/mylib_arm64-osx/lib/libtwo.a

Debug binaries were not found

Found 1 post-build check problem(s). To submit these ports to curated catalogs, please first correct the portfile:
```

I see there is a policy in the vcpkg tool itself to check or not check this, so how do we manipulate this setting from a portfile? Thanks!

**Comment by Osyotr** [comment, marked as answer]
Either set `VCPKG_POLICY_MISMATCHED_NUMBER_OF_BINARIES` inside the portfile or just copy these libs to `debug/lib`.

**Reply by kevin--** [reply]
perfect! thank you -- is there some place these policies are documented?

**Reply by Osyotr** [reply]
I don't think so...
But their names are pretty evident
https://github.com/microsoft/vcpkg/blob/master/scripts/build_info.cmake

**Reply by kevin--** [reply]
great reference... i was poking around trying to find this stuff and could not. Thanks again!
